doc/developer_guide/index.md

```diff
@@ -2,6 +2,10 @@
 
 The Panel library is a complex project which provides a wide range of data interfaces and an extensible set of plotting backends, which means the development and testing process involves a wide set of libraries.
 
+This guide describes how to install and configure the development environment fully.
+
+If you have any problems with the steps here, please reach out in the `dev` channel on [Discord](https://discord.gg/rb6gPXbdAr).
```

Comment:
What about discourse instead?

Reply:
I would argue that the `dev` channel on Discord is the most *welcoming* place to go for new contributors. Its faster to get help. You can start chatting. Its *extra friendly* to me someone live in the other end. You become a member of the *dev community*.

The only downside I can see is that the history on Discourse is much more discoverable.

Reply:
I suppose discord you need to make an account which not everyone has.